graphql_api/mutation/comment.py

```diff
@@ -6,11 +6,15 @@
 from controller.comment import manager
 
 
+def send_notification():
+    pass
```

Comment:
is this a preparation for further development?

Reply:
nope, deleted :)